pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -628,6 +628,7 @@ func (m *kubeGenericRuntimeManager) killContainersWithSyncResult(pod *v1.Pod, ru
 		} else {
 			containers = runningPod.Containers
 		}
+		m.preStopSidecars(pod, sidecars)
```

Comment:
any unit test for this PR?

Reply:
what would you suggest as a test for this? the function does not return anything, it just attempts to pre-stop the containers and logs any failures encountered when it does this.